src/endpoints/next_js/client_jsonParse.ts

```
@@ -0,0 +1,79 @@
+                                    // push all the keys to foundUrls
+                                    foundUrls.push(...keys);
+                                }
+                            } catch (e) {
+                                // Ignore errors from JSON.parse
+                            }
+                        }
+                    }
+                },
+            });
+        } catch (err) {
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)